src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/infrastructure/client/util/HomeOfficeDateFormatter.java

```diff
@@ -62,4 +62,24 @@ public static String getPersonDateOfBirth(int dayOfBirth, int monthOfBirth, int
         return hoDateOfBirth;
     }
 
+    public static String getIacDateAndTime(String homeOfficeDate) {
```

Comment:
I this as per the requirements I don't think we need to explicitely add the time factor. If time available we send the time details otherwise it is just a date. Could you please verify once?

Reply:
com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `java.time.OffsetDateTime` from String "2020-02-01": Text '2020-02-01' could not be parsed at index 10

Reply:
I get the above exception if I don't add the midnight time (when no time given)

Reply:
We are doing this in other places. We can go ahead with now but need to see if this can be removed.